### PR TITLE
Pilot Officer Slot Reduction: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Al-1ce**
## About The Pull Request

Reduces PO slots to one.
<sub>For preface, this is intended to be different from #11702, as Naani didn't originally advertise it in the way that I thought was best for the server. ...Only after they were given a shitload of dislikes did it look like something marginally reasonable. </sub>

## Why It's Good For The Game

PO is an ...interesting role, to say the least. I've got around ~110 hours on it, mostly flying CAS, which I think gives me some say in the way things work here. There is a lingering issue with having multiple pilots, and I'll explain that here.

1. Latejoin as PO
2. Tadpole is gone <sub>(Synth/CSE/FC/etc took it, these roles are allowed and encouraged to do this)</sub>
3. The other pilot is doing CAS
4. You are functionally useless and without a job. 

Imagine if we treated other roles the same way. Picture spawning in as a regular squad marine to see every gun and vendor empty. You'd either wait and scavenge around, or cryo.

That is exactly how the PO feels. I feel like this is a terrible oversight; one that I and many others have experienced. And I don't feel like rebranding the role to CAS is the right away to go about things, since it can be mind-numbingly boring at times.
### TL;DR

* Two POs roundstart leads to bloat since other roles want (and will) use the tadpole

## Changelog

:cl:
balance: PO has a singular slot.
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
People not having fun isn't a priority issue in this videogame.

**Comment by Otterwatch** [comment]
This pr is the same as Naani's in all but framing. I don't disagree with the pr, but it's still doing the same thing besides changing the name of the role. Reminder for those reading the pr comments and otherwise, ahelp if you're not having fun as a role and want to swap roles. The online admin can return you to lobby. Generally, you want to do this after just spawning or before deployment, but there are some exceptions.

**Comment by Pariah919** [comment]
CSE shouldn't have had really.

**Comment by Al-1ce** [comment]
> ahelp if you're not having fun as a role and want to swap roles

The key issue is this having to be done in the first place. I cannot think of another role where something like this happens as often as the PO -- it becomes a mechanical and design issue at that point. Players shouldn't have to ahelp to get a job-swap because their role is bloated with more people than need be.

>CSE shouldn't have tad really

The issue here is that many jobs are relying on rules that were written 3 years ago. Back when we had specialists, capture-gameplay and admins would ticket you for saying "smh" and the like. The server has changed, but much of our rules haven't.

CSE and ST are shitty in their current state and are unallowed to actually play the fucking game (shoot beno) without restrictions or getting in trouble. It's only been a recent thing where our administration has stopped caring or offered leeway, like the CSE taking the tadpole to reinforce a disk.

But that's all really moot. The inherent conflict of two Pilot Officers vying for each-others goal is unfun and creates a needless level of bloat that you won't find elsewhere. Marines, for example, do not spawn saying "dibs on condor" or something of like. 

You don't spawn in as a marine with the knowledge that there is a 50/50 chance you're not going to be guarantied to play the way you want to.  Whenever two POs spawn roundstart, that's a constant. It's not fun; I've seen it happen, I've had it happen to me, dozens of times. It's a flaw with the role innately.

>This pr is the same as Naani's in all but framing

The intent of the PR is key, especially when it comes to how roles are intended to be played. Naani wanted PO to be CAS exclusive; and for a time, I agreed with them. But that's not right, CAS can be a fucking drag. Trust me, I know better than most. 

And with that experience I can say full-heartedly that 2 POs is very overcrowded and leaves little fun for the extra-guy who missed out on the tadpole being taken by whomever and the Condor being in-use. There isn't another marine role where you're quite literally shit out of luck. 

>Okay Boo, Nanni already tried this. It is not going to happen. Please just let this idea die, maints are just going to close the PRs anyways.

The PO is overcrowded: simple as. There is no need for more than one pilot officer at any given time. Play the role for more than a couple hours find yourself stranded with nothing to do, then maybe you'll understand why I made this PR.

**Comment by Lumipharon** [comment]
This is the same PR.

Its an IC or admin issue if people are taking tad as non PO and not letting the PO do their actual job.

If you explicitly want to be doing tad stuff then you should be playing as PO, not going CSE then being upset when the actual PO wants to do it.

**Comment by Al-1ce** [comment]
least clueless maintainer

**Comment by Al-1ce** [comment]
> Its an IC or admin issue if people are taking tad as non PO and not letting the PO do their actual job.

An IC issue, ..caused by a clear mechanical oversight. I think you're just trying to give the administration team something to entertain themselves with.

> If you explicitly want to be doing tad stuff then you should be playing as PO, not going CSE then being upset when the actual PO wants to do it.

And expect the CSE to do ...what, exactly? AFK on the ship to replace the fusion-gens every fifteen minutes? Stay in the FOB and do nothing?

A lot of issues about this server stem from our maints being giga-retards and not understanding 1. this is a game - and 2. there are certain "IC issues" that can be permanently resolved with a small mechanical change.

I digress. Kill me for caring about peoples enjoyment.

**Comment by Losenis** [comment]
>Its an IC or admin issue if people are taking tad as non PO and not letting the PO do their actual job.

PO is the absolute worst role to take tadpole as. Admins can believe whatever they want, mechanically the PO should be the absolute last person that shold be taking the tadpole. Literally the only reason Tadpole PO exists is for a rookie PO to ahelp the other people out before he realizes he can just pick any other job to run tadpole with, at which point he'll just be ahelped out by the next rookie PO that'll suicide the tadpole due to, again, rookie.

A second PO slot has zero relevancy to the game and only exists to be a "mom says it's my turn" role that any other roll can fulfill, most of the time way better. The only relevancy it has is in metaganging with other people who actually have useful roles to use the tadpole with and ahelp them out. Again, ingame the second PO slot is **irrelevant**.

**Comment by Losenis** [comment]
And just for a thought experiment, do you consider it an IC issue if competent people are running tadpole, xenos die to it, a xeno waits and respawns as PO, ahelps for the tadpole up and then suicides it into the xenos?

"It won't happen", I had a PO ahelp an hour into the round trying to kick me out of the tadpole despite having had it for the whole round. There is, inevitably, zero way to tell what the intentions were and we can't read minds so we're going to be stuck at 'nonzero chance' no matter what one argues. Can't say yes, can't say no, the possibility is there and this weirdass ruling encourages it.
"The admin can just say no", then that PO will just AFK to death, cue this PR again in the future for the trillionth time.
"He can just ahelp for a respawn", how is it ok to ahelp for a respawn every time this happens till the heat death of the universe but actually fixing the issue isn't?

**Comment by Joe13413** [comment]
> And just for a thought experiment, do you consider it an IC issue if competent people are running tadpole, xenos die to it, a xeno waits and respawns as PO, ahelps for the tadpole up and then suicides it into the xenos?

my man making up actual fucking conspiracy theories to get PO slots reduced lol

**Comment by Al-1ce** [comment]
> my man making up actual fucking conspiracy theories to get PO slots reduced lol

It's happened before. Enough for it to be a sort of stereotype.

**Comment by FIXTHERADBOXPLEASE** [comment]
That's a Marine player kicking off a synth for perfectly legitimate reasons, not someone switching sides to sabotage the Tadpole(banable offense.)
